packages/universal-components/src/RangeDatePicker/AnimatedDayItemArrow.js

```diff
@@ -34,35 +34,48 @@ const RenderArrows = ({ scale }) => (
 );
 
 export default class AnimatedDayItemArrow extends React.Component<Props> {
-  static scale: Animated.Value = new Animated.Value(0);
+  static scale: Animated.Value;
   static animating: boolean;
+  static numberOfInstances: number;
 
   static handleAnimation() {
-    AnimatedDayItemArrow.scale.setValue(0);
-
-    Animated.sequence([
-      Animated.timing(AnimatedDayItemArrow.scale, {
-        ...SHARED_ANIMATION_CONFIG,
-        toValue: 1,
-      }),
-      Animated.timing(AnimatedDayItemArrow.scale, {
-        ...SHARED_ANIMATION_CONFIG,
-        toValue: 0,
-      }),
-    ]).start(() => {
-      if (AnimatedDayItemArrow.animating) {
+    if (AnimatedDayItemArrow.scale === undefined) {
+      AnimatedDayItemArrow.scale = new Animated.Value(0);
+    }
+    if (
+      AnimatedDayItemArrow.scale &&
+      AnimatedDayItemArrow.scale._value === 0 &&
+      AnimatedDayItemArrow.numberOfInstances !== 0
```

Comment:
Animation freezes when picker is opened for second time. It looks like because of this condition.
![anim-freeze](https://user-images.githubusercontent.com/2660330/56119349-44fc4180-5f6c-11e9-9f39-8be1337ba377.gif)